### ISSUE TITLE
Search Optgroup

### ISSUE THREAD

**Issue by hugofabricio** [lcdsantos/jQuery-Selectric#199]
How to text search in optgroup?

**Comment by robs-scc** [comment]
I try to do a quick keyboard navigation to a specific option within an optgroup, but I'm unable to.  For example, if there is a list of options (just number text) within optgroups, I can't use the keyboard (number keys) to jump to a specific number.  Up and down arrow keys work though.
